pkg/api/norman/customization/clusterregistrationtokens/import.go

```diff
@@ -6,15 +6,22 @@ import (
 	"github.com/gorilla/mux"
 	"github.com/rancher/norman/types"
 	"github.com/rancher/norman/urlbuilder"
+	v3 "github.com/rancher/rancher/pkg/generated/norman/management.cattle.io/v3"
 	"github.com/rancher/rancher/pkg/image"
 	schema "github.com/rancher/rancher/pkg/schemas/management.cattle.io/v3"
 	"github.com/rancher/rancher/pkg/settings"
 	"github.com/rancher/rancher/pkg/systemtemplate"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func ClusterImportHandler(resp http.ResponseWriter, req *http.Request) {
+type ClusterImport struct {
+	Clusters v3.ClusterInterface
+}
+
+func (ch *ClusterImport) ClusterImportHandler(resp http.ResponseWriter, req *http.Request) {
 	resp.Header().Set("Content-Type", "text/plain")
 	token := mux.Vars(req)["token"]
+	clusterID := mux.Vars(req)["clusterId"]
```

Comment:
hmm How does this work? ClusterId is not a query parameter on the URL?

Reply:
we updated that as well, https://github.com/rancher/rancher/pull/31381/files#diff-68120b5b2bc57e41b2dee9f83f5022c4487dd82cb597c132152c58ffc4286e5bR169